### PR TITLE
feat(droppable): x, y position and target within event

### PR DESCRIPTION
**Pull request by soyuka**
Hi, this is a suggestion to add some informations to the DragEvent. 

Should I continue working on this and add tests etc. or is it something you don't want?

### CODE REVIEW

**Comment by mattlewis92** [comment]
> Hi, this is a suggestion to add some informations to the DragEvent.
> 
> Should I continue working on this and add tests etc. or is it something you don't want?

Sure thing, I think it makes sense to add! Out of curiosity, what's your use case?

**Comment by soyuka** [comment]
Hi, we have a context of containers that can contain other containers. This helps us place "placeholders" according to the position on the parent container (above, below or inside).

File: projects/angular-draggable-droppable/src/lib/droppable.directive.ts [hunk 1]
```diff
@@ -121,10 +122,10 @@ export class DroppableDirective implements OnInit, OnDestroy {
           }
         );
 
-        let currentDragDropData: any;
+        let currentDragDropData: DragEvent;
```
Comment (mattlewis92) [inline projects/angular-draggable-droppable/src/lib/droppable.directive.ts:125]: nit: maybe we could rename this from `currentDragDropData` to `currentDragEvent`?

**Review by mattlewis92 — CHANGES_REQUESTED** [review]
LGTM! Just had one small suggestion then this should be g2g. Thanks for your contribution!

**Comment by soyuka** [comment]
done and rebased

**Review by mattlewis92 — APPROVED** [review]
LGTM, thank you! 🙌

**Pull request merged**